suites/nautilus/upgrades/upgrade_3.x_ceph-disk_to_4.x_lvm.yaml

```diff
@@ -93,6 +94,7 @@ tests:
     desc: run rados bench for 360 - normal profile
 - test:
     name: ceph ansible purge
+    polarion-id: CEPH-83571498
     module: purge_cluster.py
     desc: Purge ceph cluster
     abort-on-fail: True
```

Comment:
i dont think we need recreate cluster here,pls remove it , unless for any specific use case later we can add, same applies to  suite in #127

Reply:
@XDsubhash I have made the changes can you please review it.

Reply:
remove it frm the suite in #127

Reply:
okay